pkg/k8s_tap_service.go

```diff
@@ -23,6 +25,11 @@ import (
 	"github.com/turbonomic/kubeturbo/pkg/discovery/monitoring/master"
 )
 
+const (
+	usernameFilePath = "/etc/opsmgr-credentials/username"
```

Comment:
I actually thought, we are going to use envfromsecret instead of a mounted volume

Reply:
ok, we can drop this